protobuf/answer.proto

```diff
@@ -34,12 +34,17 @@ message ConceptMapGroup {
     repeated ConceptMap concept_maps = 2;
 }
 
-message Number {
-    string value = 1;
-    // We use a string to contain the full representation of a number
+message Numeric {
+    oneof value {
+        int64 long_value = 1;
+        double double_value = 2;
+        NaN nan_value = 3;
```

Comment:
I think we can just use a `bool` here rather than defining a separate message - like we do elsewhere in protocol (for example `done` and `continue` in `Transaction.Res`)
```suggestion
        bool nan = 3;
```

Reply:
Fixed in https://github.com/graknlabs/protocol/pull/102/commits/d859686566b66e3099c435bee09d8d48f0f67df0